biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -120,6 +120,7 @@
       "File_Parsing.ipynb\r\n",
```

Comment:
<p>You use <code>str.find</code>  in this cell but <code>str.contains</code>  in the next. Is it intentional for these to be different?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>No. That was an error. <code>str.contains</code>  is the correct version.</p>